packages/geo/src/coordinates/latlon/internal/parse.test.ts

```diff
@@ -0,0 +1,66 @@
+// __private-exports
+/*
+ * Copyright 2024 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { expect, it, describe } from 'vitest';
+
+import { parse } from './parse';
+
+describe('parser', () => {
+  it.each`
```

Comment:
Minor nit with the `it.each` approach: if we wanted to use these pairs in other tests, for mocks, for fixtures, etc we would have to copy paste this block around. Putting the pairs (value, expectation) into a fixture means we can use it all over the place if needed.

Reply:
Yeah, I can see that. However, it is very nice to be able to see the input an expected output at a glance. Is this something we want to change?

Reply:
Looks like we can maybe leverage `it.for` https://vitest.dev/api/#test-for

```
const tests = [
  ['0 0', '0 / 0', [],
  ['.01 .02', '0.01 / 0.02', []
]

it.for(tests)(
  '%i | %i | %i', 
  [a, b, errors] => {
    // expect
  }
);

```

That way we can pass around `tests` still if needed for other things

Reply:
Either would work. Do you want me to make that change now or can it wait till we need to reuse test cases? The only thing I hang onto is that the template string gets auto-formatted for column layouts #lazy-dev.